tests/integration/tests/test_gateway.py

```diff
@@ -47,13 +50,26 @@ def test_gateway(session_instance: harness.Instance):
             "kubectl",
             "get",
             "service",
-            "cilium-gateway-my-gateway",
-            "-o=jsonpath='{.spec.ports[?(@.name==\"port-80\")].nodePort}'",
```

Comment:
Does replacing this with Python (below) give any advantages? If we keep it as jsonpath, the code is 10 lines shorter.

Reply:
We've got to grab all the services and check for one with "my-gateway" in the name to make it more general for cilum/contour...